_posts/2022-09-23-discrete-events-simulation-at-providence-outpatient-care-services-covid-19-recovery-scheduling/des-for-covid-19-recovery-scheduling.Rmd

```diff
@@ -0,0 +1,80 @@
+description: |
+  A introduction to Descrete Events Simulations (DES) in healthcare with an overview of the use of DES for modelling outpatient services appointments at Providence Healthcare.  With the Covid-19 lockdown, there was a large backlog of appointments that needed to be filled.  Providence Outpatient Services explored the use of DES to evaluate increasing Staffing numbers, changing clinic hours and booking more appointments while maintaining social distancing requirements in their clinical and waiting areas.
+  
+author:
+  - name: Kevin Wang, Neil Mistry
+    url: https://www.linkedin.com/in/mingkun-wang/, https://www.linkedin.com/in/neilmistry/
+date: 2022-09-23
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+# Discrete Events Simulation at Providence Outpatient Care & Services Covid-19 Recovery scheduling
+
+## Intro
+
+Discrete Events Simulations (DES) are stochastic models which allow us to simulate a real world system or process.  It is modelled by a system as a (discrete) sequence of events in time.  Each event occurs at a particular instant in time and marks a change of state in the system, e.g. An arrival of a patient, or the delivery of a container of parts.  
+This type of model is particularly useful in simulating queues or systems which have a sequence of steps (particularly manufacturing, or call centers).  Because queuing models become increasingly complex to solve, as we increase the number of nodes, DES may be the only solution to model these systems.  DES can be modelled with simple routing rules and probabilities of events occurring.
+
+These simulations are useful to test hypothetical changes in the system.  For instance, say you wish to add an extra teller to a grocery store, what would be the impact to customer wait times?  Is it worth the additional staffing cost?  What if we only added this teller during peak customer times?  What is the cost to benefit then?
+DES has long been proven successful in manufacturing, services, and supply chain sectors in solving complex resourcing and efficiency questions.  With growing concern on increasing costs to provide healthcare, and perceived lack of efficiency, there is growing usage of DES in healthcare.
+DES is commonly used to test schedules (for operating rooms, nursing, emergency department staffing, or other resourcing), test equipment purchase decisions (how many IV pumps are required to provide full coverage to the hospital?), space requirements (how many patient rooms, waiting chairs, do we need in this ED?) and even managing patient admissions and scheduling procedures.
+
+At Providence, with the second Covid-19 pandemic dwindling, we wanted to test schedules to cut down the growing backlog of outpatient appointments required to fill while ensuring safe social distancing practices within the hospital.
```

Comment:
Was this project completed around the second wave of the pandemic? Would that be towards the end of 2020? Early 2021?

Reply:
Yes it was.